### PR TITLE
{bio}[foss/2021b] CodingQuarry v2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/c/CodingQuarry/CodingQuarry-2.0-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+toolchainopts = {'openmp': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['CodingQuarry_v%(version)s.tar.gz']
+checksums = ['1198afbf7cebcf0975c5b20d92b7a2dd6d956072fcde6e86fdce6aeae4842504']
+
+buildopts = 'CFLAGS="$CFLAGS"'
+
+files_to_copy = [
+    (['CodingQuarry', 'CufflinksGTF_to_CodingQuarryGFF3.py'], 'bin'),
```
Comment (branfosj) [inline easybuild/easyconfigs/c/CodingQuarry/CodingQuarry-2.0-GCC-11.2.0.eb:19]: A python file copied to bin and no Python dependency? There are more python scripts in `QuarryFiles/scripts` and all have issues:
* The Python scripts have `#!/usr/bin/python` as a first first line.
* Requires Python2 or some alterations: `print 'run: \n\npython CufflinksGTF_to_CodingQuarryGFF3.py transcripts.gtf > out.gff3\n'`
* BioPython dependency needed: `from Bio import SeqIO`

**Review by branfosj — DISMISSED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3102.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/d08ad8c3db4f7b572e904d15576c1c95 for a full test report.

File: easybuild/easyconfigs/c/CodingQuarry/CodingQuarry-2.0-GCC-11.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/c/CodingQuarry/CodingQuarry-2.0-GCC-11.2.0.eb]: fixed in 7f91101 + 7a983ccad1

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/4b8a5ee33739fb47a620823c4a775d6c for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**